core/src/main/java/com/vml/es/aem/acm/core/code/Arguments.java

```diff
@@ -146,6 +146,36 @@ public void integerNumber(String name, Closure<IntegerArgument> options) {
         add(argument);
     }
 
+    public void color(String name) {
+        color(name, null);
+    }
+
+    public void color(String name, Closure<ColorArgument> options) {
+        ColorArgument argument = new ColorArgument(name);
+        GroovyUtils.with(argument, options);
+        add(argument);
+    }
+
+    public void integerRange(String name) {
+        integerRange(name, null);
```

Comment:
it does not make sense to have range without options set, remove that signature and similar cases